src/Models/Command/CommandContext.cs

```diff
@@ -45,4 +53,13 @@ public T GetService<T>() where T : class
     {
         return _serviceProvider.GetRequiredService<T>();
     }
+    
+    /// <summary>
+    /// Sends real-time results to the client during long-running operations
+    /// </summary>
+    /// <param name="response">The response containing real-time results to send</param>
+    public void SendRealtimeResults(CommandResponse response)
+    {
+        _mcpServer?.SendRealtimeResults(response);
```

Comment:
I feel like I am missing something because I don't see SendRealtimeResults defined in IMcpServer

Reply:
ive pushed the files i missed. the mcp server does support streaming please look at https://mcp-framework.com/docs/Transports/http-stream-transport/